cmd/installer/goods/support/host-support-bundle.tmpl.yaml

```diff
@@ -287,7 +287,7 @@ spec:
       args: [ "sysinfo" ]
   - copy:
       collectorName: installer-logs
-      path: /var/log/embedded-cluster/*.log
+      path: /var/log/*/*.log
```

Comment:
We don't want to collect logs from all directories in /var/log, we should template this and use the logs directory variable to render it.